src/backend/distributed/operations/auto_shard_split.c

```diff
@@ -0,0 +1,370 @@
+#include "postgres.h"
+#include "libpq-fe.h"
+#include "executor/spi.h"
+#include "distributed/lock_graph.h"
+#include "distributed/coordinator_protocol.h"
+#include "distributed/metadata_cache.h"
+#include "distributed/metadata_utility.h"
+#include "distributed/multi_logical_replication.h"
+#include "postmaster/postmaster.h"
+#include "distributed/distribution_column.h"
+#include "utils/builtins.h"
+#include "distributed/shard_split.h"
+
+
+PG_FUNCTION_INFO_V1(citus_auto_shard_split_start);
+
+int MaxShardSize = 102400;
+
+/*
+ * Struct to store all the information related to
+ * a shard.
+ */
+typedef struct ShardInfoData
+{
+	int64 shardsize;
```

Comment:
minor, we usually do more camelCasing, e.g. shardSize

Reply:
done for all the variables